### PR TITLE
chore: add volunteer SattyamSamania

### PR DESCRIPTION
**Pull request by SattyamSamania**

### CODE REVIEW

File: data/users/SattyamSamania.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Sattyam Samania",
+    "bio": "Frontend Developer",
+    "image": "https://avatars.githubusercontent.com/u/67833888?s=400&u=7f15bce07fcea153c36a222a598de3df5fbae3e0&v=4",
```
Comment (codewithvoid) [inline data/users/SattyamSamania.json:4]: use simpler url https://avatars.githubusercontent.com/u/67833888?v=4

Comment (SattyamSamania) [reply data/users/SattyamSamania.json:4]: > use simpler url https://avatars.githubusercontent.com/u/67833888?v=4

Hey @codewithvoid thanks for the suggestions I have made another PR. Please Check it.

**Review by SattyamSamania — COMMENTED** [review]
I have made some changes. Feel free to tell me if any corrections are required

File: data/users/SattyamSamania.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Sattyam Samania",
+    "bio": "Frontend Developer",
+    "image": "https://avatars.githubusercontent.com/u/67833888?v=4",
+    "categories": [
+      {
+        "slug": "web-development",
+        "topics": ["CSS", "Tailwind CSS","Bootstrap"],
+        
+        "slug": "Programming",
```
Comment (codewithvoid) [inline data/users/SattyamSamania.json:10]: snake case for slug  - `programming`

File: data/users/SattyamSamania.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Sattyam Samania",
+    "bio": "Frontend Developer",
+    "image": "https://avatars.githubusercontent.com/u/67833888?v=4",
+    "categories": [
+      {
+        "slug": "web-development",
+        "topics": ["CSS", "Tailwind CSS","Bootstrap"],
```
Comment (codewithvoid) [inline data/users/SattyamSamania.json:8]: remove space

**Comment by codewithvoid** [comment]
Perfect. Thanks for making the changes.

**Pull request merged**